tests/framework/installer/ceph_installer.go

```diff
@@ -152,27 +154,21 @@ func (h *CephInstaller) startAdmissionController() error {
 	return nil
 }
 
-// CreateRookOperatorViaHelm creates rook operator via Helm chart named local/rook present in local repo
-func (h *CephInstaller) CreateRookOperatorViaHelm(chartSettings string) error {
-	// creating clusterrolebinding for kubeadm env.
-	h.k8shelper.CreateAnonSystemClusterBinding()
-
-	// create the operator namespace before the admission controller is created
-	if err := h.k8shelper.CreateNamespace(h.settings.OperatorNamespace); err != nil {
-		return errors.Errorf("failed to create namespace %s. %v", h.settings.Namespace, err)
-	}
-	if err := h.startAdmissionController(); err != nil {
-		return errors.Errorf("failed to start admission controllers: %v", err)
+func (h *CephInstaller) waitForToolboxPod(namespace string) (*string, error) {
```

Comment:
Why return a pointer instead of the string?
```suggestion
func (h *CephInstaller) waitForToolboxPod(namespace string) (string, error) {
```